aws-auth-cognito/README.md

```diff
@@ -0,0 +1,145 @@
+## User Acceptance Testing Guide
```

Comment:
This is a weird way to start the plugin README.

Reply:
Yeah this was just for John Pignata during development - removing